src/d2lfetch-simple-cache.js

```diff
@@ -1,4 +1,35 @@
-import parseCacheControl from 'parse-cache-control';
```

Comment:
This 3rd party library was CommonJS and I just couldn't get it to work in ESM-land, so I re-created the parts we need (check for `no-cache`, `no-store` and extract the `max-age`) below: